src/Controls/src/Core/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -71,5 +72,29 @@ public static partial class AppHostBuilderExtensions
 
 		public static IMauiHandlersCollection AddMauiControlsHandlers(this IMauiHandlersCollection handlersCollection)
 			=> handlersCollection.AddHandlers(DefaultMauiControlHandlers);
+
+		internal static MauiAppBuilder ConfigureImageSourceHandlers(this MauiAppBuilder builder)
+		{
+			builder.ConfigureImageSources(services =>
+			{
+				services.AddService<FileImageSource>(svcs => new FileImageSourceService(svcs.GetService<IImageSourceServiceConfiguration>(), svcs.CreateLogger<FileImageSourceService>()));
+				services.AddService<FontImageSource>(svcs => new FontImageSourceService(svcs.GetRequiredService<IFontManager>(), svcs.CreateLogger<FontImageSourceService>()));
+				services.AddService<StreamImageSource>(svcs => new StreamImageSourceService(svcs.CreateLogger<StreamImageSourceService>()));
+				services.AddService<UriImageSource>(svcs => new UriImageSourceService(svcs.CreateLogger<UriImageSourceService>()));
+			});
+
+			return builder;
+		}
+
+		internal static MauiAppBuilder RemapForControls(this MauiAppBuilder builder)
```

Comment:
- Centralize this call so that I can call it from the unit tests
- Moved this code into Controls because it doesn't have any dependencies on Compatibility